.github/workflows/deploy-staging.yml

```diff
@@ -70,6 +70,7 @@ jobs:
           eventId: ${{ steps.trigger_deploy.outputs.eventId }}
           crtFile: ${{ secrets.STAGING_CLIENT_CERTIFICATE }}
           keyFile: ${{ secrets.STAGING_CLIENT_PRIVATE_KEY }}
+          timeout: 900000
```

Comment:
is this in milliseconds or seconds?

Reply:
it's milliseconds, we watch the execution for max 15 minutes